extensions-api/src/main/scala/pl/touk/nussknacker/engine/ProcessingTypeConfig.scala

```diff
@@ -1,14 +1,10 @@
 package pl.touk.nussknacker.engine
 
 import com.typesafe.config.Config
-import net.ceedubs.ficus.readers.ValueReader
-
-import java.io.File
-import java.net.URL
 
 case class ProcessingTypeConfig(
     deploymentManagerType: String,
-    classPath: List[URL],
+    classPath: List[String],
```

Comment:
Isn't it an API's class? Maybe we should put down an entry in MigrationGuide? Why we change URL to String? At the end you convert these strings to URI..